main.py

```
@@ -0,0 +1,166 @@
+    # Set up colors if terminal supports them
+    if curses.has_colors():
+        curses.start_color()
+        curses.init_pair(1, curses.COLOR_WHITE, curses.COLOR_BLACK)  # Default
+        curses.init_pair(2, curses.COLOR_GREEN, curses.COLOR_BLACK)  # Landing zone
+        curses.init_pair(3, curses.COLOR_RED, curses.COLOR_BLACK)    # Warning
+    
+    # Clear screen
+    stdscr.clear()
+
+
+def main(stdscr):
```

Ruff (0.8.2)

`main` is too complex (11 > 10)

(C901)